packages/react-components/react-tags/src/components/TagGroup/renderTagGroup.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
```

Comment:
```suggestion
/** @jsxRuntime classic */
/** @jsx createElement */
import { createElement } from '@fluentui/react-jsx-runtime';
```
This is odd, I'm pretty sure I've added this to `create-component` tool 🤔

Reply:
I created the component (last week?) and cherry-picked that commit. Then I had some problem with getSlots so I swapped to getSlotsNext. So probably back then you changes wasn't in master?